doc/tech/approach.md

```diff
@@ -80,27 +80,19 @@ To address the concern areas detailed above the specific topics listed below wil
 
 This section introduces some networking related principles that should be followed during development and definition of Networking scopes for Reference Model, Reference Architecture 01 (OpenStack), Reference Architecture 02 (K8s) and down to Reference Implementations and Reference Conformance test suites under CNTT.
 
-•	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
+-	**Abstraction**: A standardized layer between Virtualisation Layer domains and the Networking Hardware Layer resource pool hiding the implementation details of the Networking Hardware.
 > **Note:** Abstraction may be deployed in many different ways e.g. depending on target use case requirements, workload characteristics, different algorithm implementions of pipeline stages and available platforms. This includes Network Functions running on server nodes with or without programmable hardware acceleration, or networking support functions running on programmable network switches in the network.
```

Comment:
Not sure how abstraction can "be deployed". "Network Functions running on server nodes with or without programmable hardware acceleration ..." is not abstraction.

What is the purpose of this Note? Maybe then we can word it appropriately.

Reply:
@pgoyal01 I do not know. I did not write it. I agree that abstractions can not be deployed. What if I change it to implemented?

Reply:
@CsatariGergely Can we first decide the purpose of this note -- what is it that we want to convey? Then we can word accordingly. Thanks

There are other terms such as "pipeline stages" (of what) that all need to be addressed.

Reply:
I'm happy to discuss this note, but this pr was supposed to be about something else. ...
For me th enote does not add any clarity. We could even remove it in a different pr.